docs/README.md

```diff
@@ -2018,17 +2098,15 @@ socket.
 ```
 URL: "<httpURL>/graphql"
 ```
+
 ```graphql
 mutation {
-  ChangeCart(input: {
-    cartId: "demo"
-    sku: "ABC_02"
-    quantity: 3
-  })
+  ChangeCart(input: { cartId: "demo", sku: "ABC_02", quantity: 3 })
```

Comment:
I don't think these changes improve readability...

Reply:
It was the format on save linter 🤦